drivers/bus/isapnp/CMakeLists.txt

```diff
@@ -19,3 +19,4 @@ set_module_type(isapnp kernelmodedriver)
 add_importlibs(isapnp ntoskrnl hal)
 add_pch(isapnp isapnp.h SOURCE)
 add_cd_file(TARGET isapnp DESTINATION reactos/system32/drivers NO_CAB FOR all)
+add_registry_inf(isapnp_reg.inf)
```

Comment:
Probably need to mark the boot driver also here: https://github.com/reactos/reactos/blob/master/boot/bootdata/txtsetup.sif#L103

Just add the "x" here:
```inf
isapnp.sys   = 1,,,,,,x,4,,,,1,4
```